# KirimOffsetUdp.msg
# Message untuk mengirim data offset posisi via UDP

float32 posisi_x_offset
float32 posisi_y_offset
float32 sudut_w_offset
